packages/edit-navigation/src/components/layout/sidebar.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * WordPress dependencies
+ */
+import { BlockInspector } from '@wordpress/block-editor';
+import { useDispatch } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+import {
+	ComplementaryArea,
+	store as interfaceStore,
+} from '@wordpress/interface';
+import { __ } from '@wordpress/i18n';
+import { cog } from '@wordpress/icons';
+
+const BLOCK_INSPECTOR_SCOPE = 'core/edit-navigation';
```

Comment:
I think we can move these two to `/utils/constants.js`

Reply:
👍 I did that. I've also moved `constants.js` out of utils and into its own folder (`constants/index.js`). Slightly nicer than having to traverse up to root and then down into utils.